static/src/build.js

```diff
@@ -12,50 +12,51 @@ const routes = {}
 console.log(chalk.magenta('Found these pages:'))
 Object.keys(routes).map(r => console.log(r))
 
-Object.keys(routes).forEach(async function(route) {
-  console.log(chalk.magenta(`Attempting ${chalk.blue('React.createElement')} on ${route}`))
-
-  // Handle un-connect() wrapped components
-  // console.log(routes[route])
-  // console.log(Object.keys(routes[route]()))
-  let component
-  try {
-    component = routes[route]().default
-  } catch(e) {
-    console.log('component err, ', e)
-  }
-
-  if (typeof(component) === 'function') {
+let ps = Object.keys(routes).forEach(route => () => {
```

Comment:
my style is to always wrap params unless its a one-liner map,
i think redux middleware written that way is less readable lol

Reply:
resolved